currentRangesTest.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import currentRanges
+
+class CurrentRangesTest(unittest.TestCase):
+  def test_count_min(self):
+    self.assertTrue(currentRanges.count_min([4,4,5,5,4,8]) == [4,3])
+  def test_remove_min(self):
+    self.assertTrue(currentRanges.remove_min([4,4,5,5,4,8]) == [5,5,8])
+  def test_def_range_count(self):
```

Comment:
Try using the test function name to describe the functionality.
In this case, instead of `test_def_range_count`, you could word it as `test_counts_in_range_excluding_single_values`
These function names can be long... we don't write code to call them.